README.md

```diff
@@ -1,596 +1,493 @@
+// deployer address
+const deployerAddress = '0xaabb1122....................';
 
-let ephemeralKey = '0xaabb1122....................';
+// Worker address
+const worker = '0xaabb1122....................';
 
-let facilitatorAddress = '0xaabb1122....................';
+// Relayer address
+const relayer = '0xaabb1122....................';
 
-let passphrase = 'some passphrase.....';
+// It's needed for unlock account before doing transaction.
+const passphrase = 'some passphrase.....';
 
-// some other constants
+// Other constants
 const gasPrice = '0x12A05F200';
-const gas = 8000000;
-```
+const gas = '7500000';
 
-###### ABI and BIN provider
-
-OpenST.js comes with an in-built abi-bin provider for managing abi(s) and bin(s).
-
-The abiBinProvider provides developers with following abi(s) and bin(s):
-
-* [MockToken](https://github.com/OpenSTFoundation/mosaic-contracts/blob/v0.9.3-rc1/contracts/SimpleToken/MockToken.sol) (an EIP20 contract with name 'MockToken')
-* [TokenRules](https://github.com/OpenSTFoundation/openst-contracts/blob/v0.9.4/contracts/TokenRules.sol) (a registry of rule contracts and the conduit for transfers)
-* [TokenHolder](https://github.com/OpenSTFoundation/openst-contracts/blob/v0.9.4/contracts/TokenHolder.sol) (a multi-sig wallet that can hold tokens)
-* [TransferRule](https://github.com/OpenSTFoundation/openst-contracts/blob/v0.9.4/contracts/TransferRule.sol) (a simple transfer rule contract)
-
-```js
-// Get the MockToken ABI
-let abiBinProvider = openST.abiBinProvider();
-let mockTokenAbi = abiBinProvider.getABI('MockToken');
 ```
 
-#### Adding accounts
-One can add accounts to web3 wallet or to our custom signer service. Please choose only ONE of the options described below.
-
-Remember to add all of deployerAddress, organizationAddress, wallet1, wallet2, ephemeralKey and facilitatorAddress to use the functionality given in this readme.
+### Deploy Organization contract
 
-###### Option 1: Add accounts to web3 wallets
-For adding to web3 wallet, there are 2 ways. We can add using the keystore file OR by private key. If you choose this option, you could also choose to connect to an [infura endpoint](https://infura.io/docs) rather than running a full geth node.
-
-Detailed documentation on adding accounts can be found here: https://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html
+An Organization contract serves as an on-chain access control mechanism by assigning roles to a set of addresses.
 
 ```js
-// Here is a sample helper method for developers to add an account to the wallet using keystore content
-// Read more about web3.eth.accounts.decrypt here: https://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html#id22
-let addToWalletByKeystoreContent = function (encryptedPrivateKey, password) {
-  let account = web3.eth.accounts.decrypt(encryptedPrivateKey, password);
-  web3.eth.accounts.wallet.add(account);
+// Deploy Organization contract
+const Mosaic = require('@openstfoundation/mosaic.js');
```

Comment:
we should highlight on top of the readme that for organization you need this dependency

Reply:
Updated in "Creating an OpenST object" section.